### PR TITLE
Final updation

### PR DESCRIPTION
**Pull request by Abhimanyu1311**

### CODE REVIEW

File: src/Components/Navbar.js [hunk 1]
```diff
@@ -12,15 +13,19 @@ function Navbar() {
...
+            console.error('Error parsing cart data:', error);
         }
     };
-    const totalAmount = cartItems.reduce((sum, item) => sum + item.quantity, 0)
+
+    const totalAmount = cartItems.reduce((sum, item) => sum + item.quantity, 0);
+
     useEffect(() => {
         handleTotal();
+        const intervalId = setInterval(handleTotal, 100);
```
Comment (ankur1778) [inline src/Components/Navbar.js]: why did you add this?

File: src/Pages/Login.js [hunk 2]
```diff
@@ -28,12 +28,15 @@ export default function Login() {
...
+
             if (data.accessToken) {
                 localStorage.setItem("accessToken", data.accessToken);
                 resetForm();
                 navigate("/");
-            } 
+            }
+            else{
+                alert("Abey Yaaar!! Dekh kr password daal")
+            }
```
Comment (ankur1778) [inline src/Pages/Login.js]: Abe yrr! dekh alert daal

**Review by ankur1778 — CHANGES_REQUESTED** [review]
please resolve comments

File: src/Components/Navbar.js (hunk 1 above)
Comment (Abhimanyu1311) [reply src/Components/Navbar.js]: To handle the cart count

File: src/Pages/Login.js (hunk 2 above)
Comment (Abhimanyu1311) [reply src/Pages/Login.js]: Invalid k liye vo notification vale bht sare daal kr dekhe h ni chl rhe pr koi b

Comment (Abhimanyu1311) [reply src/Pages/Login.js]: To last me ye dalla h

Comment (Abhimanyu1311) [reply src/Pages/Login.js]: Hta du ky

File: src/Components/Navbar.js (hunk 1 above)
Comment (ankur1778) [reply src/Components/Navbar.js]: No need to add setInterval. Please remove this and handle without this

File: src/Pages/Login.js (hunk 2 above)
Comment (ankur1778) [reply src/Pages/Login.js]: For invalid you have throw error. not set any alert.

File: src/Components/Navbar.js (hunk 1 above)
Comment (Abhimanyu1311) [reply src/Components/Navbar.js]: okay

**Review by ankur1778 — APPROVED** [review]
